todos.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from uuid import UUID
+from fastapi import FastAPI, HTTPException
+
+from pydantic import BaseModel
+
+app = FastAPI()
+
+todos = []
+
+class Task(BaseModel):
+  name: str
+  description: str
+  tag: str
+  created_time: str
+  updated_time: str
+  task_id: UUID
+
+@app.get("/")
+def root():
+  return {"Welcome to your To Do list"}
+
+@app.get("/todos")
+def read_todos():
+  return todos
+
+@app.get("/todos/{id}")
+def read_task(id: int):
+  return{"task": todos[id]}
+
+@app.post("/todos/")
+def create_task(task: Task):
+  task.created_time = datetime.now()
+  todos.append(task)
+  return task
+
+@app.put("/todos/{id}")
+def update_task(id: int, task: Task):
+  itemToUpdate = todos[id]
```

Comment:
@grantsydney - This is good. Now how can this function handle an invalid id that is not in the index without throwing an error?. How can we let the user know that they have provided an invalid id?